consumer/retry_counter_test.go

```diff
@@ -0,0 +1,44 @@
+package consumer
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestRetryCounter(main *testing.T) {
```

Comment:
Odd naming of the testing argument? Usually this is `t`?

Reply:
Yes, I simply follow the existing testing pattern.

Reply:
:+1: Makes sense. Didn't think of that.